extensions/notebook/package.json

```diff
@@ -125,6 +125,26 @@
 				"key": "Ctrl+Shift+T",
 				"when": "notebookEditorVisible"
 			}
+		],
+        "notebook.languagemagics": [
+			{
+				"magic": "lang_python",
+				"language": "python",
+				"executionTarget": null,
+				"kernels": ["sql"]
+			},
+			{
+				"magic": "lang_r",
+				"language": "r",
+				"executionTarget": null,
+				"kernels": ["sql"]
+			},
+			{
+				"magic": "lang_java",
+				"language": "java",
+				"executionTarget": null,
+				"kernels": ["sql"]
+			}
```

Comment:
These are the SQL execution engines that are supported by us in 2019 / shipped in box. Long-term would be great to dynamically detect, since in theory external folks can register any language they want now. But that'll require more thought.

If and when we add a "Spark" execution target then we'd do Scala, Python and R